src/fustat/definitions.cairo

```diff
@@ -118,6 +118,52 @@ namespace secp256r1 {
     const MIN_ONE_D3 = 0x0;
 }
 
+namespace bandersnatch {
```

Comment:
generate this code automatically with the https://github.com/keep-starknet-strange/garaga/blob/cd329cc873b83841b0828080f6e6db95cae87ef8/hydra/definitions.py#L53-L75 to_cairo_zero method